### PR TITLE
Visibility fix for internal members

### PR DESCRIPTION
**Pull request by vbuberen**
## :page_facing_up: Context
Fix for #179 

## :pencil: Changes
- Updated visibility modifiers
- Changed fun for starting TransactionActivity, since after merging PR with `ViewModel` this fun was reverted to state before #137 
- Fixed typo, which appeared after #146 
- Replaced deprecated calls for JsonParser

### CODE REVIEW

File: library/build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ android {
     compileSdkVersion rootProject.compileSdkVersion
 
     compileOptions {
-        kotlinOptions.freeCompilerArgs += ['-module-name', "com.github.ChuckerTeam.Chucker.library-no-op"]
```
Comment (cortinico) [inline library/build.gradle]: ++

Comment (ColtonIdle) [reply library/build.gradle]: Unfamiliar with what's going on here. Can someone explain?

Comment (cortinico) [reply library/build.gradle]: Sure. This was a fix I introduced for #134 (in #146). I accidentally copy and pasted the same line for `library` and `library-no-op` without adapting the module name to the proper artifact name.

**Comment by vbuberen** [comment]
@cortinico This PR is ready for review.

**Pull request merged**